concert-tickets/src/main/java/halatsiankova/javafromscratch/validator/NullValidatorProcessor.java

```diff
@@ -0,0 +1,31 @@
+package halatsiankova.javafromscratch.validator;
+
+import halatsiankova.javafromscratch.annotation.NullableWarning;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+public class NullValidatorProcessor {
```

Comment:
Good implementation of checkNullFields method but according to homework task it should be called during object creation not after. To make this possible you could place this logic into parent entity or some interface and call it in the end of constructor methods